### PR TITLE
this works too

### PR DESCRIPTION
**Pull request by amazingalek**

### CODE REVIEW

File: OWML.Common/Menus/IModMainMenu.cs [hunk 1]
```diff
@@ -2,9 +2,9 @@
 {
     public interface IModMainMenu : IModOWMenu
     {
-        IModTabbedMenu OptionsMenu { get; }
-        IModButton OptionsButton { get; }
-        IModButton QuitButton { get; }
+        new IModTabbedMenu OptionsMenu { get; }
+        new IModButton OptionsButton { get; }
+        new IModButton QuitButton { get; }
```
Comment (amazingalek) [inline OWML.Common/Menus/IModMainMenu.cs:7]: removes the warnings 🤷

File: OWML.Common/Menus/IModMenu.cs [hunk 2]
```diff
@@ -20,8 +20,6 @@ public interface IModMenu
         IModButton GetTitleButton(string title);
         IModPromptButton GetPromptButton(string title);
 
-        IModButton AddButton(IModButton button);
-        IModButton AddButton(IModButton button, int index);
```
Comment (amazingalek) [inline OWML.Common/Menus/IModMenu.cs:24]: didn't seem to need these?

File: OWML.ModHelper.Menus/ModButtonBase.cs [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 namespace OWML.ModHelper.Menus
 {
-    public abstract class ModButton : IModButtonBase
+    public abstract class ModButtonBase : IModButtonBase
```
Comment (amazingalek) [inline OWML.ModHelper.Menus/ModButtonBase.cs:9]: less confusing for me

File: OWML.Common/Menus/IModMenu.cs (hunk 2 above)
Comment (TAImatem) [reply OWML.Common/Menus/IModMenu.cs:24]: try with SpawnPoint mod

File: OWML.ModHelper.Menus/ModButtonBase.cs (hunk 3 above)
Comment (TAImatem) [reply OWML.ModHelper.Menus/ModButtonBase.cs:9]: sure, why not

**Pull request merged**